eth-types/src/evm_types.rs

```diff
@@ -54,12 +54,24 @@ impl ProgramCounter {
 #[derive(Default, Clone, Copy, Eq, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
 pub struct Gas(pub u64);
 
+impl fmt::Display for Gas {
```

Comment:
I have the feeling that all these wrapper types `ProgramCounter` `Gas` and `GasCost` add verbosity to the code but don't provide many benefits; and using `usize` and `u64` directly will lead to cleaner code.  We would avoid having to do things like `Gas(0)` instead of just `0`.  Also avoid doing `step.gas.0` instead of `step.gas`. What do you think about this?
Note: this comment is unrelated to this PR.

Reply:
I feel you strongly on this too! We should remove ProgramCounter, Gas, and GasCost structs in other PR.

Reply:
Created #1404 to track this.